src/main/java/game/jdbc/picturepoker/GameDAO.java

```diff
@@ -91,4 +103,23 @@ public Game update_string(String attribute, String data, Game dto){
             throw new RuntimeException(e);
         }
     }
+
+    // This is the second time I had to program this. Richard, please don't go around reverting my changes willy-nilly.
```

Comment:
Sorry about this. 
Github features do include looking at past code, so if you ever want to revert code (revert reverts, or something like that), you can just re access previous commits, and physically copy and paste the text (if you don't want to deal with git revert, and all that messy stuff). 

Sorry about the mess up, hopefully this doesn't happen again.